compiler/tests-compiler/js_parser_printer.ml

```diff
@@ -445,6 +445,34 @@ let%expect_test "string template" =
     s=
      `asd ${ /*<< 7 20>>*/ f(`space ${test} space`,32)} te`; |}]
 
+let%expect_test "string template" =
+  (* GH#1017 *)
+  print
+    ~report:true
+    ~compact:false
+    {|
+({key:"from",
+ value:
+   function from(field,get)
+     {if(!get)get=function get(x){return x;};
+      return this.compute([field],function(state){return get(state.field(field));});}})
+ |};
+
+  [%expect
+    {|
+    /*<< 2 0>>*/ ({key:"from",
+     value:
+     function get(field,get)
+      { /*<< 5 6>>*/ if(! get)
+         /*<< 5 14>>*/ get
+        =
+        function get(x){ /*<< 5 34>>*/ return x /*<< 5 18>>*/ };
```

Comment:
This test is failing for me. I believe this test is demonstrating the bug, rather than the fix.